src/app/games/checkers/common/CheckersFailure.ts

```diff
@@ -0,0 +1,29 @@
+import { Localized } from 'src/app/utils/LocaleUtils';
+
+export class CheckersFailure {
+
+    public static readonly CANNOT_GO_BACKWARD: Localized = () => $localize`You cannot go backward with normal pieces!`;
+
+    public static readonly CANNOT_SKIP_CAPTURE: Localized = () => $localize`You must capture when it is possible!`;
+
+    public static readonly MUST_FINISH_CAPTURING: Localized = () => $localize`You must finish this capture!`;
+
+    public static readonly THIS_PIECE_CANNOT_MOVE: Localized = () => $localize`This piece cannot move!`;
+
+    public static readonly CAPTURE_STEPS_MUST_BE_DIAGONAL: Localized = () => $localize`A capture should be diagonal. Look at the indicators to help you!`;
+
+    public static readonly NORMAL_PIECES_CANNOT_MOVE_LIKE_THIS: Localized = () => $localize`Normal pieces cannot move like this!`;
+
+    public static readonly NORMAL_PIECES_CANNOT_CAPTURE_LIKE_THIS: Localized = () => $localize`Normal pieces cannot capture like this!`;
```

Comment:
un peu vague celui-ci et le précédent, on veut préciser ou pas ? (Le pattern était, il me semble « Vous avez fait ceci. Pour capturer, faites cela » non ?)

Reply:
La version "vous avez fait ça. faites ceci" fait un toast vachement long non ?

Reply:
ça fait long c'est sûr, mais peut-être en plus court y'a moyen d'avoir un peu plus de précision que "like this"? Sinon TPR !